### DISCUSSION TITLE
UFW Problem

### DISCUSSION THREAD

**Discussion by sh4rk21** [pterodactyl/panel#2844] [Support] [answered]
Hi, I have a problem with UFW, all my port are manually opened for my gaming server, but when i try to connect it doesn't work and when i see on internet is not opened. 

I sought on internet and i found its probably a conflict between UFW and DOCKER it's possible ?

Someone have any issues ?

Thank you

**Comment by parkervcp** [comment]
The issue is likely that you need to open the ports in ufw. Pterodactyl and Wings do no touch your firewall rules. Docker does edit iptables rules though.

**Reply by sh4rk21** [reply]
Thank you for your help, i have already done like this :
` ufw allow in on pterodactyl0 to 172.18.0.1 port <LOCALHOST_PORT> proto tcp`
 in UDP/TCP But it doesn't work.

**Comment by TekExplorer** [comment, marked as answer]
what ip are you using for your allocations? if you are using 127.0.0.1 or 172.18.0.1 your servers wont be internet-facing.

also, some hosts, like google cloud, can have external firewalls, so check that

**Reply by sh4rk21** [reply]
Hi Tek Explorer, I was using another ip, 172.18.0.1 it was docker ip. I have delete all my vm and reinstall all and it works, thank you for your help.

**Reply by TekExplorer** [reply]
please make sure to mark a solution as answered if you can :)

**Comment by Feniksovich** [comment]


**Reply by DomiiBunn** [reply]
They don't
